addOns/automation/src/main/java/org/zaproxy/addon/automation/AuthenticationData.java

```diff
@@ -172,12 +197,11 @@ public AuthenticationData(Object data, AutomationProgress progress) {
             if (!StringUtils.isEmpty(method)
                     && !validMethods.contains(method.toLowerCase(Locale.ROOT))) {
                 progress.error(
-                        Constant.messages.getString("automation.error.env.auth.type.bad", data));
+                        Constant.messages.getString(AUTOMATION_ERROR_ENV_AUTH_TYPE_BAD, data));
             }
             for (Entry<String, Object> entry : parameters.entrySet()) {
                 switch (entry.getKey()) {
-                    case PARAM_PORT:
-                    case PARAM_LOGIN_PAGE_WAIT:
+                    case PARAM_PORT, PARAM_LOGIN_PAGE_WAIT:
```

Comment:
Noise.

Reply:
Do you mean the CONST declarations? (For all the noise comments)

Reply:
https://github.com/zaproxy/zap-extensions/pull/6112#discussion_r1928324092

Reply:
Please be more specific.

Reply:
Oh, I think I see.